packages/esm-ward-app/src/hooks/useObs.ts

```diff
@@ -0,0 +1,22 @@
+import useSWR from 'swr';
+import { type Observation } from '../types';
+import { type Link, openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+
+type QueueEntrySearchCriteria = Record<'patient', string> | Record<'encounter', string> | Record<'q', string>;
+
+// TODO: right now, the obs endpoint can only take in one of "patient", "encounter" and "q' as param
+// We should make it able to query by patient AND concept type
```

Comment:
Wow, that Obs endpoint is definitely in need of work.  Let's ticket that.

Reply:
What @mogoodrich suggested works (thanks!). I'm changing it to pass in just patient and concept